src/core/lib/channel/channelz.h

```diff
@@ -268,10 +268,15 @@ class SocketNode : public BaseNode {
 // Handles channelz bookkeeping for listen sockets
 class ListenSocketNode : public BaseNode {
  public:
-  ListenSocketNode();
+  // ListenSocketNode takes ownership of host.
+  ListenSocketNode(char* host, int port);
```

Comment:
Suggest making the `host` parameter a `UniquePtr<char>`, so that the ownership transfer is inherent in the interface and can be enforced by the compiler.

Reply:
Done